mvpa2/base/externals.py

```diff
@@ -438,7 +445,7 @@ def __check(name, a='__version__'):
         exec "v = %s.%s" % (name, a)
         # it might be lxml.etree, so take only first module
         versions[name.split('.')[0]] = SmartVersion(v)
-    except Exception, e:
+    except Exception as e:
```

Comment:
This syntax is not supported in Python 2.5.; the website mentions that Python 2.5 is supported.